src/context.jl

```diff
@@ -442,7 +445,7 @@ end
                 customtypes = tupcat(customtypes, nonstandardtype(col.type))
             end
         end
-        checkinvalidcolumns(types, "types", ncols, names)
+        validate && checkinvalidcolumns(types, "types", ncols, names)
```

Comment:
I think it would be useful to mention that you can pass `validate=false` in the `checkinvalidcolumns` error just so people know what to do immediately if they hit the error.

Reply:
yep, it's already added (up on L104), but let me know if it could be clearer; it currently reads:
```diff
-            (0 < k <= ncols) || throw(ArgumentError("invalid column number provided in `$argname` keyword argument: $k. Column number must be 0 < i <= $ncols as detected in the data"))
+            (0 < k <= ncols) || throw(ArgumentError("invalid column number provided in `$argname` keyword argument: $k. Column number must be 0 < i <= $ncols as detected in the data. To ignore invalid columns numbers in `$argname`, pass `validate=false`"))
         else
-            Symbol(k) in names || throw(ArgumentError("invalid column name provided in `$argname` keyword argument: $k. Valid column names detected in the data are: $names"))
+            Symbol(k) in names || throw(ArgumentError("invalid column name provided in `$argname` keyword argument: $k. Valid column names detected in the data are: $names. To ignore invalid columns names in `$argname`, pass `validate=false`"))
```